src/test/java/io/vertx/ext/couchdb/utils/IntegrationTest.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * IntegrationTest annotation
+ */
+package io.vertx.ext.couchdb.utils;
```

Comment:
`io.vertx.ext.couchdb.testannotations;` ?

Reply:
Sure, that would be a better package name